src/core-services/cart/simpleSchemas.js

```diff
@@ -829,10 +828,6 @@ export const Cart = new SimpleSchema({
     type: CartAddress,
     optional: true
   },
-  "sessionId": {
-    type: String,
-    optional: true
-  },
```

Comment:
We probably shouldn't remove this one yet, unless we already have a migration that removes it from the database. Otherwise old carts will fail validation next time we try to update them. We don't yet have a way of doing migrations in 3.0